jme3-lwjgl3/src/main/java/com/jme3/system/lwjgl/LwjglWindow.java

```diff
@@ -562,6 +578,7 @@ protected void deinitInThread() {
 
         destroyContext();
         super.internalDestroy();
+        glfwTerminate();
```

Comment:
I just wonder here if we can't/shouldn't arrange it that the restart could call deinitInThread() and initInThread instead of manually re-creating the context.

I think all that would be required is moving the glfwTerminate post deinitInThread, right?
I don't know if that works by the API.

Reply:
Well, I tried it just now. Seems to actually fix #1013 . But also causes some side effects. Cursor is somehow gone as well as some other elements and all inputs... Need to investigate this a bit more...